runtime/doc/lua.txt

```diff
@@ -841,10 +841,18 @@ vim.call({func}, {...})					*vim.call()*
             vim.fn[func]({...})
 
 vim.cmd({cmd})                                          *vim.cmd()*
-        Invokes an Ex command (the ":" commands, Vimscript statements).
+        Executes multiple lines of Vimscript at once. It's It is an alias 
```

Comment:
Looks good now apart from the stray `It's` here. ;]

Reply:
fixed. I've no idea why how it was even there ;}